InnerEye/ML/utils/checkpoint_handling.py

```diff
@@ -20,17 +20,21 @@
 from InnerEye.Common import fixed_paths
 
 
+@dataclass
+class CheckpointPathsAndEpoch:
+    """
+    Holds the path path to a checkpoint and the checkpoint epoch.
+    """
+    epoch: int
+    checkpoint_paths: Optional[List[Path]]
```

Comment:
nitpicking: Do we need to distinguish between None and an empty list?

Reply:
Done - checkpoint_paths will be a list with 0 or more elements.